libsrc/atari/sound.s

```diff
@@ -0,0 +1,39 @@
+;
+; Mariano Domínguez
+; 2022-12-4
+;
+; atari lib
+;
+        .include "atari.inc"
+        .export         __sound
+        .import         popa
+; play sound, arguments: voice, pitch, distortion, volume. same as BASIC
+.proc   __sound
+        sta STORE2      ;save volume
```

Comment:
somehow this comment got lost before...

Why don't you use tmpX zero-page variables? Please use them (e.g. tmp1 and tmp2)